### ISSUE TITLE
Flatten NPC is immediately removed after being applied on 4.1.2.

### ISSUE THREAD

**Issue by Solidor777** [velara/pf2e-flatten#6]
Reproed on a fresh world with all other modules disabled. 
![Flatten Without Level broken](https://user-images.githubusercontent.com/14845045/191643436-13a3616c-5c3b-4604-99bb-effb7edc7ba7.gif)

**Comment by Solidor777** [comment]
Console Warning

```
commons.js:1696 
        
       Error: You are accessing the NPCPF2e#data object which is no longer used. Since V10 the Document class and its contained DataModel are merged into a combined data structure. You should now reference keys which were previously contained within the data object directly.
    at logCompatibilityWarning (commons.js:1685:19)
    at NPCPF2e._logV10CompatibilityWarning (commons.js:6612:14)
    at get data [as data] (commons.js:6512:24)
    at Object.condition (bundle.js:102:19)
    at ContextMenu.render (foundry.js:55435:65)
    at HTMLLIElement.<anonymous> (foundry.js:55373:19)
    at HTMLElement.dispatch (jquery.min.js:2:43064)
    at y.handle (jquery.min.js:2:41048)
logCompatibilityWarning @ commons.js:1696
_logV10CompatibilityWarning @ commons.js:6612
get data @ commons.js:6512
condition @ bundle.js:102
render @ foundry.js:55435
(anonymous) @ foundry.js:55373
dispatch @ jquery.min.js:2
y.handle @ jquery.min.js:2
commons.js:1696 
        
       Error: You are accessing the NPCPF2e#data object which is no longer used. Since V10 the Document class and its contained DataModel are merged into a combined data structure. You should now reference keys which were previously contained within the data object directly.
    at logCompatibilityWarning (commons.js:1685:19)
    at NPCPF2e._logV10CompatibilityWarning (commons.js:6612:14)
    at get data [as data] (commons.js:6512:24)
    at Object.condition (bundle.js:81:19)
    at ContextMenu.render (foundry.js:55435:65)
    at HTMLLIElement.<anonymous> (foundry.js:55373:19)
    at HTMLElement.dispatch (jquery.min.js:2:43064)
    at y.handle (jquery.min.js:2:41048)
 ```

**Comment by velara** [comment]
It was working on v10 285 and PF2e 4.0.5. I've managed to reproduce your error on V10 286 and PF2e 4.1.2 and I've found the source of the error, but I don't have time to test my solution for bugs tonight. Hopefully I'll be able to push a fix tomorrow.

**Comment by velara** [comment]
Fixed in V0.4.1
